Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'danger-dangermattic', '~> 1.0'
 gem 'fastlane', '~> 2.216'
 gem 'fastlane-plugin-appcenter', '~> 2.1'
 gem 'fastlane-plugin-sentry', '~> 1.14'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 11.0', '>= 11.0.2'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 12.2'
```

Comment:
I have checked [our Migration Guide from 11.x to 12.0](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/MIGRATION.md#from-11x-to-1200) and didn't see any changes listed there that would apply to what's used in PCiOS's `Fastfile`. But feel free to double-check to be sure I didn't overlooked something!

Reply:
Ah, just noticed something the `CHANGELOG.md` mentions but `MIGRATION.md` doesn't: `ios_current_branch_is_hotfix` has been removed, so `finalize_release` will fail.

I had the same problem recently on WPiOS and [had to add a similar function](https://github.com/wordpress-mobile/WordPress-iOS/pull/23674/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R129).

Reply:
Ah, nice catch. We should update the `MIGRATION.md` so that it's not missed by migrations of other apps!

Reply:
Oh, there's also `create_release`, now it should be `create_github_release`.

Reply:
> `ios_current_branch_is_hotfix` has been removed, so finalize_release will fail.

This should now be fixed in 264cf8e. Thanks for linking to your WPiOS implementation for inspiration!

> ~Oh, there's also `create_release`, now it should be `create_github_release`.~ NVM, the action isn't really being called 😅 it's just a `create_release` flag.

Yeah, almost made the same mistake when I searched the Fastfile for the various methods in `MIGRATION.md` 😅